app/js/containers/Seller/SellerContactContainer.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {Component, Fragment} from 'react';
+import PropTypes from 'prop-types';
+import SellerContact from '../../components/Seller/SellerContact';
+import seller from "../../features/seller";
+import {connect} from "react-redux";
+
+class SellerContactContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      nickname: props.nickname,
+      contactCode: props.contactCode
+    };
+    this.validate(props.nickname, props.contactCode);
+    props.footer.onPageChange(() => {
+      props.setContact({nickname: this.state.nickname, contactCode: this.state.contactCode});
+    });
+  }
+
+  validate(nickname, contactCode) {
+    if (contactCode && nickname) {
+      this.props.footer.enableNext();
+    } else {
+      this.props.footer.disableNext();
+    }
+  }
+
+  changeContactCode = (contactCode) => {
+    this.validate(this.state.nickname, contactCode);
+    this.setState({contactCode});
+  };
+
+  changeNickname = (nickname) => {
+    this.validate(this.state.contactCode, nickname);
+    this.setState({nickname});
+  };
+
+  render() {
+    return (
+      <Fragment>
```

Comment:
You don't need fragment if you return only 1 component.
Also we should probably try to find what are the components that are reusable SellerContact instead of doing a 1-1 with the container
(we can probably refactor that later)